workspaces/liferay-customer-workspace/quickstart/docker-compose.yaml

```diff
@@ -28,4 +28,11 @@ services:
         ports:
             - 8080:8080
         volumes:
-            - ../build/docker/client-extensions:/opt/liferay/osgi/client-extensions
\ No newline at end of file
+            - ../build/docker/client-extensions:/opt/liferay/osgi/client-extensions
+    mail:
+        container_name: liferay-customer-workspace-mail
+        image: maildev/maildev
+        ports:
+            - 1080:1080
+            - 1025:1025
```

Comment:
can you sort ports in numerical order

Reply:
re-pushed, sorry the port number didn't all work so was messing with it